### ISSUE TITLE
Using SceneNN train ESAM

### ISSUE THREAD

**Issue by JingShuju** [xuxw98/ESAM#28]
Thank you very much for your remarkable work; it is truly outstanding. In your research, you utilized the SceneNN dataset for ESAM evaluation. Is it possible to use the SceneNN-formatted dataset for model training as well? I am eagerly looking forward to your response. thank you again!

**Comment by xuxw98** [comment]
Yes. You can follow our dataset preperation to format scenenn training set the same as scannet, then run the training script by modifying dataset related config.
